platform/entanglement/test/services/CopyServiceSpec.js

```diff
@@ -266,6 +285,20 @@ define(
                     });
                 });
 
+                it("throws an expection when performed on invalid inputs", function () {
```

Comment:
It should probably also indicate that it throws an error, rather than an expection.